kratos/sources/variables.cpp

```diff
@@ -60,6 +60,7 @@
 #include "includes/kratos_flags.h"
 
 namespace Kratos {
+typedef Node<3> NodeType;
```

Comment:
i believe that this file should not be modified as a part of this PR

Reply:
Ok